operator/watchers/ciliumendpoint/utils.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ciliumendpoint
+
+import (
+	"fmt"
+
+	ciliumApiV2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
+)
+
+// HasCEWithIdentity returns true or false if the Cilium Endpoint store has
+// the given identity.
+func HasCEWithIdentity(identity string) bool {
+	if CiliumEndpointStore == nil {
+		return false
+	}
+	ces, _ := CiliumEndpointStore.IndexKeys(identityIndex, identity)
+
+	return len(ces) != 0
+}
+
+// HasCE returns true or false if the Cilium Endpoint store has the endpoint
+// with the given name.
+func HasCE(ns, name string) (*ciliumApiV2.CiliumEndpoint, bool, error) {
```

Comment:
The function name and description isn't matching. It should be `GetCE`.

Reply:
Done